src/Symfony/Component/Serializer/Mapping/Loader/AttributeLoader.php

```diff
@@ -43,10 +43,6 @@ class AttributeLoader implements LoaderInterface
         Context::class,
     ];
 
-    public function __construct()
```

Comment:
this might break child classes that call parent::__construct
not likely, but not worth taking the risk either

Reply:
Indeed, forgot about that case. That's reverted, no other constructor is removed in the rest of the PR